js/ui/bind_handlers.js

```diff
@@ -37,6 +38,11 @@ module.exports = function bindHandlers(map, options) {
     el.addEventListener('dblclick', onDblClick, false);
     el.addEventListener('contextmenu', onContextMenu, false);
 
+    function onMouseOut(e) {
+        map.stop();
```

Comment:
Let's remove this line. By default, animated zooms/pans should not stop if the cursor leaves the canvas.